app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,85 @@
+<%= stylesheet_link_tag 'recipes/show', media: 'all' %>
+<div class="recipe__info">
+  <h2 class="recipe__title"><%= @recipe.name %></h2>
+  <div class="recipe__vision">
+    <p>Preparation time: <%= @recipe.preparation_time %></p>
+    <% if current_user == @recipe.user %>
+    <p>
+      Public
+      <label class="recipe__switch">
+        <input type="checkbox" id="recipe_public" <%= @recipe.public ? 'checked' : '' %>>
+        <span class="slider"></span>
+      </label>
+    </p>
+    <% end %>
+  </div>
+  <p>Cooking time: <%= @recipe.cooking_time %></p>
+  <p><%= @recipe.description %></p>
+  <div class="recipe__action">
+    <%= link_to 'Generate Shopping list' , recipe_general_shopping_list_path(@recipe.id), class:"recipe__button" %>
+    <% if current_user == @recipe.user %>
+      <%= link_to 'Add ingredient', new_recipe_recipe_food_path(@recipe.id), class:"recipe__button" %>
+    <% end %>
+  </div>
+</div>
+
+<div class="recipe__detail">
```

Comment:
- [ ] We can not add any ingredient that also us to see if the shopping list and food ingredients work as expected. Please fix it in order to make your app works as expected :+1: 

![ingred](https://user-images.githubusercontent.com/6615394/225974991-09cf8496-d64c-4ef6-b37e-52d916fa8c2a.png)